workflows/steps/services/uma_export/go.mod

```diff
@@ -5,3 +5,5 @@ go 1.23.0
 replace github.com/GoogleChrome/webstatus.dev/lib => ../../../../lib
 
 replace github.com/GoogleChrome/webstatus.dev/lib/gen => ../../../../lib/gen
+
+require github.com/GoogleChrome/webstatus.dev/lib v0.0.0-00010101000000-000000000000
```

Comment:
I don't have the best answer for this.

The changes to the go.mod and go.sum files are normally done by the Go tool chain.

Here's my hypothesis though:
When we normally add an external package, the tool chain adds the package with a version number or a hash to those files. But since I'm referring to a local module, it may be generating this version automatically as part of the multi module go workspace. So the zeros and ones may be part of some internal implementation detail for handling local modules.